guard.cpp

```diff
@@ -18,7 +18,8 @@ void guardList()
     std::cout << "ID       | ERROR    |  Type  | Path " << std::endl;
     for (const auto& elem : records)
     {
-        if (elem.errType != GARD_Reconfig)
+        if ((elem.errType != GARD_Reconfig) &&
+            (elem.recordId != GUARD_RESOLVED))
```

Comment:
Not seeing the update in commit message related to this change.

Reply:
Done.